Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.15 as builder
+WORKDIR /go/src/github.com/openshift/coredns-ocp-dnsnameresolver
+COPY . .
+
+RUN git clone https://github.com/openshift/coredns /go/src/github.com/openshift/coredns
+RUN GO111MODULE=on GOFLAGS=-mod=mod hack/add-plugin.sh /go/src/github.com/openshift/coredns
+
+WORKDIR /go/src/github.com/openshift/coredns
+RUN GO111MODULE=on GOFLAGS=-mod=vendor go build -o coredns .
```

Comment:
It's used in the openshift/coredns Dockerfile.openshift file, so I used it here as well to build coredns.
https://github.com/openshift/coredns/blob/b6a657d656bef2631c4675a819ce25a4662b0978/Dockerfile.openshift#L4

Reply:
Yeah, I see. The last commit in the coredns' Dockerfile was made 5 years ago. I think about this time the go modules were introduced. `GO111MODULE` is not needed now as the module mode is default since go 1.13 or 1.14.